src/libraries/sendtohelixhelp.proj

```diff
@@ -233,13 +232,32 @@
     </ItemGroup>
   </Target>
 
+  <PropertyGroup Condition="'$(IsXUnitLogCheckerSupported)' == 'true' and '$(BuildTargetFramework)' == '$(NetCoreAppCurrent)'">
+    <_ShouldSetXUnitLogCheckerEnvVar>1</_ShouldSetXUnitLogCheckerEnvVar>
+    <HelixPreCommands Condition="'$(TargetOS)' == 'windows'">
+      $(HelixPreCommands);
+      set __TestArchitecture=$(TargetArchitecture)
```

Comment:
```suggestion
      set __TestArchitecture=$(TargetArchitecture);
```

Reply:
@hoyosjs I actually missed a semicolon here if I'm going to follow the right pattern.